css/styles.css

```diff
@@ -0,0 +1,126 @@
+* { 
+  box-sizing: border-box; 
+}  
+
```

Comment:
@akallbold try using a `body` selector to apply general styles to your stylesheet (since everything will inherit from the top down) then specify which element(s) should  have additional styles applied. This way you write less code.

Reply:
I had in my notes that we should use * for this, but changed "*" to "body".

Reply:
@akallbold Your notes are correct, for `box-sizing: border-box` you should keep the rule as it was with the `*` as the selector.